server/backend_api/process/views.py

```diff
@@ -1,16 +1,36 @@
-from django.shortcuts import render
+from django.shortcuts import render, redirect
 from django.http import HttpResponse
+from django.views.decorators.csrf import csrf_exempt
 from django.template import loader
 from django.core.files.storage import FileSystemStorage
 from asgiref.sync import async_to_sync
 from channels.layers import get_channel_layer
 import string
 import random
+from process.models import User
 
+
+@csrf_exempt
 def home(request):
+    username = request.POST.get('username')
+    password = request.POST.get('password')
+    for user in User.objects.all():
+        if user.username == username and user.password == password:
+            return render('load_file.html')
     return render(request, 'pages/home.html')
 
 
+@csrf_exempt
+def register(request):
```

Comment:
Tego nie kminie, bo w widoku html po naciśnięciu przycisku zawsze musi wysłać POST a on jakby otrzymywał GET i przekierowuje do register znowu i to jeszcze register/register xD

Reply:
Jak najbardziej wysyła się POST. Problem jest w tym, że na samym końcu funkcji przetwarzającej POST request dajesz
```
return redirect('pages/home.html')
```
podałeś w nim ścieżkę lokalną, nas interesuje URL dostępny publicznie. Nie ma takiego adresu jak `localhost/pages/home.html` więc strona zwraca GET zawierający błąd